tests/ctest/CMakeLists.txt

```diff
@@ -37,6 +37,11 @@ target_link_elektra (test_cmerge elektra-merge)
 
 target_link_elektra (test_sha-256 elektra-ease)
 
+target_link_libraries (test_changetracking elektra-kdb)
+
+target_link_libraries (test_diff elektra-kdb)
+target_link_libraries (test_diffprivate elektra-kdb)
```

Comment:
```suggestion
target_link_elektra (test_changetracking elektra-kdb)

target_link_elektra (test_diff elektra-kdb)
target_link_elektra (test_diffprivate elektra-kdb)
```

Reply:
In my case the problem was that the linker was not able to find the xfconf library so this case may be a bit different. However, I would try to replace use these macros. The test `test_backends` e.g. includes `kdb.h` already and it seems there was no issue on the past.